schema/lending/collateral.sql

```diff
@@ -0,0 +1,204 @@
+CREATE TABLE lending.collateral_event (
```

Comment:
I suggest: lending.collateral_change. In theory lending.collateral_add would make sense since negative amount for add would be remove but I don't think that would be very intuitive for the user.

Reply:
Also remember to change file name